packages/react-table/src/components/Table/DraggableCell.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import GripVerticalIcon from '@patternfly/react-icons/dist/js/icons/grip-vertical-icon';
+import { Button } from '@patternfly/react-core/dist/js/components/Button/Button';
+
+export interface DraggableCellProps {
+  id: string;
+  className?: string;
+  onClick?: (event: React.MouseEvent<HTMLButtonElement, MouseEvent>) => void;
+  'aria-label'?: string;
+}
+
+export const DraggableCell: React.FunctionComponent<DraggableCellProps> = ({
+  className = '',
+  onClick,
+  'aria-label': ariaLabel,
+  id,
+  ...props
+}: DraggableCellProps) => (
+  <Button
+    id={id}
+    variant="plain"
+    className={className}
+    type="button"
+    aria-label={ariaLabel || `Draggable row draggable button`}
```

Comment:
@jessiehuff  does this make sense for default aria-label?

Reply:
Jessie is out - can you think of an alternative?

Reply:
For what its worth, DataList has no default aria-label, but its example code uses "Reorder" as it's `aria-label`, paired with `aria-labelledby` with the list item id.

Reply:
I think it is ok to leave as is and get Jessie's feedback once she returns.